### DISCUSSION TITLE
Creating a Device type for a device that is half width or shelf mounted

### DISCUSSION THREAD

**Discussion by Fjord-of-the-RIngs** [netbox-community/netbox#5970] [Q&A] [answered]
I'm in the pre-deployment phase with Netbox - I'm just getting my feet wet before my company decides if it's a tool we can use.  We use some devices that are smaller than a typical rack mounted device that live in the rack either racked side by side or mounted to a rack shelf.  How would I accurately represent that when creating the device type.  Is there a way to manually add device dimensions? 

- For reference the devices is question are networked video transmitters from Crestron
- Model: HD-MD-400
This is just one specific example but a lot of the networked devices used in the AV industry are sometimes half rack mount and or shelf mounted.

**Comment by Fjord-of-the-RIngs** [comment]
Another example of this would be an Extron DMP 64 which is 1U high, half rack wide - Meaning It occupies 1RU of space but Multiple devices can also occupy that same vertical RU space.

**Comment by xkilian** [comment, marked as answer]
See #2467  I am not sure it is a very satisfactory answer. We have the same all kinds of custom equipment that are the same. Create a parent 1U 2 BAY Shelf and put the devices in it.

**Reply by Fjord-of-the-RIngs** [reply]
That'll have to do

**Reply by Fjord-of-the-RIngs** [reply]
The parent devices hold child device in different bays.  How can I make it so that the child devices are either represented or documented in the same location as the parent?

**Reply by DanSheps** [reply]
A bay can be arbitrary.  Left, Right, 1,2,34, etc.

**Reply by Fjord-of-the-RIngs** [reply]
I see that now,  This answer helped.
